src/Validation/Validation.php

```diff
@@ -783,6 +783,13 @@ public static function decimal($check, $places = null, $regex = null)
         $decimalPoint = $formatter->getSymbol(NumberFormatter::DECIMAL_SEPARATOR_SYMBOL);
         $groupingSep = $formatter->getSymbol(NumberFormatter::GROUPING_SEPARATOR_SYMBOL);
 
+        // There are two types of non-breaking spaces - we inject a space to account for human input
+        if ($groupingSep == "\xc2\xa0" || $groupingSep == "\xe2\x80\xaf") {
```

Comment:
It would be good to expand the tests for `decimal()` to cover this new logic so we don't regress in the future.